repos.d/crux.yaml

```diff
@@ -11,7 +11,7 @@
   family: crux
   ruleset: crux
   minpackages: {{minpackages}}
-  update_period: 6h
+  update_period: 2h
```

Comment:
Just remove it, please. As far as I remember, the reason for 6h was fetch problems which resulted in timeouts and blocked other updates, that doesn't seem to be the case now, all updates (both 304 and 200) are under 1 sec. Default update interval is 10 minutes (that's minimum, in practice it's still couple of hours) and we don't want to make it artificially bigger.

Reply:
Ah, I didn't know about that. I removed the line entirely.

Reply:
Just for a bit of clarification the 6h was chosen because that's how often the file on the CRUX side was updated at the time.